beacon-chain/forkchoice/doubly-linked-tree/forkchoice.go

```diff
@@ -249,9 +252,21 @@ func (f *ForkChoice) updateBalances(newBalances []uint64) error {
 					return ErrNilNode
 				}
 				if currentNode.balance < oldBalance {
-					return errInvalidBalance
+					f.store.proposerBoostLock.RLock()
+					log.WithFields(logrus.Fields{
+						"NodeRoot":                   fmt.Sprintf("%#x", bytesutil.Trunc(vote.currentRoot[:])),
+						"OldBalance":                 oldBalance,
+						"NodeBalance":                currentNode.balance,
+						"NodeWeight":                 currentNode.weight,
+						"ProposerBoostRoot":          f.store.proposerBoostRoot,
+						"PreviousProposerBoostRoot":  f.store.previousProposerBoostRoot,
+						"PreviousProposerBoostScore": f.store.previousProposerBoostScore,
+					}).Debug("node with invalid balance, setting it to zero")
```

Comment:
I don't think this log is necessary. What action do you expect users to take from it? If we make a case for this, there are many more edge cases to log around fork choice.

If you feel strongly about adding this, please make sure the log is in the proper format. e.g., not cap and root is formatted